assets/vue/components/PageBlocked.vue

```diff
@@ -54,21 +62,40 @@ export default {
       incidents: null,
       groupFlavors: true,
       matchText: '',
+      groupNames: '',
       refreshUrl: '/app/api/blocked'
     };
   },
   computed: {
     matchedIncidents() {
+      var results = [];
       if (this.matchText) {
-        return this.incidents.filter(incident => {
+        results = this.incidents.filter(incident => {
           if (String(incident.incident.number).includes(this.matchText)) return true;
           for (const pack of incident.incident.packages) {
             if (pack.includes(this.matchText)) return true;
           }
           return false;
         });
+      } else {
+        results = this.incidents;
+      }
+      if (this.groupNames) {
+        return results.filter(incident => {
+          for (let key in incident.update_results) {
```

Comment:
I would just replace all uses of `in` with `of`.

Reply:
Was able to do that on
```
for (const groupName of Object.values(groupNamesList)) {
```
but the same thing don't work on
```
for (let key in incident.update_results) {
```
because I need the key and not the value, please advice if also that `let .. in ` should be changed in some way. (Commit coming in a minute)

Reply:
```
for (const key of Object.keys(incident.update_results)) {
```